README.md

```diff
@@ -1,9 +1,27 @@
-# matrix
-
-a simple SSL-enabled server/client written in `asyncio` + `ssl`
-
-## Current Project Structure
-
+```markdown
```

Comment:
Formatting fix: Why wrapping these in backticks? This should be normal title and a small body paragraph.